apps/users/handlers.py

```diff
@@ -276,24 +389,37 @@ async def read_students_list(university_id: int, faculty_id: Union[int, None] =
     }
 
 
-@users_router.delete("/{university_id}/students/", response_model=JSENDOutSchema, tags=["SuperAdmin dashboard"])
+@users_router.delete("/{university_id}/students/",
+                     name="delete_student",
+                     response_model=JSENDOutSchema,
+                     summary="Delete university student",
+                     responses={
+                         200: {"description": "Successful delete university student response"},
+                         422: {"model": JSENDFailOutSchema, "description": "ValidationError"}
+                     },
+                     tags=["SuperAdmin dashboard"])
 async def delete_student(university_id: int, delete_student: DeleteStudentIn, auth=Depends(get_current_user)):
     query = delete(Student).where(Student.student_id == delete_student.student_id)
 
     await database.execute(query)
-    # TODO: in response key data has empty dict value, not like it's discribed
+    # TODO: in response key data has empty dict value, not like it's described
     return {
         "data": {
             "student_id": delete_student.student_id
         },
-        "message": f"Deleted student with id {delete_student.student_id}"
+        "message": f"Deleted student with id {delete_student.student_id}",
+        "code": http_status.HTTP_202_ACCEPTED
```

Comment:
For a DELETE request: HTTP 200 or HTTP 204 should imply "resource deleted successfully".

HTTP 202 can also be returned by either operation and would imply that the instruction was accepted by the server, but not fully applied yet.

Reply:
204?

Reply:
yes

Reply:
sorry change it to 2000